Compile errors with Arduino

Trying to compile throws
```
Arduino/libraries/epdiy/src/epd_driver/board/epd_board_v6.c:2:10: fatal error: board/epd_board_v6.h: No such file or directoryResolveLibrary(board/epd_board_v6.h)
  -> candidates: []
 #include "board/epd_board_v6.h"


```
I know the Arduino precompiler is bizarre and I have often had path issues like this. 
It is something I never really solved.

--- a/src/epd_driver/board/epd_board_v6.c
+++ b/src/epd_driver/board/epd_board_v6.c
@@ -1,5 +1,5 @@
 #include "epd_board.h"
-#include "board/epd_board_v6.h"
+#include "../include/board/epd_board_v6.h"
 
 #include "esp_log.h"
 #include "../display_ops.h"

